packages/core/test/hotkeys/hotkeysTests.tsx

```diff
@@ -167,19 +171,151 @@ describe("Hotkeys", () => {
+                comp = mount(<TestComponent />, { attachTo });
+                const node = ReactDOM.findDOMNode(comp.instance());
+
+                dispatchTestKeyboardEvent(node, eventName, "1");
+                expect(getLocalSpy(eventName).called).to.be.true;
+
+                dispatchTestKeyboardEvent(node, eventName, "2");
+                expect(getGlobalSpy(eventName).called).to.be.true;
+            });
+
+            it("triggers local and global hotkeys on keyup", () => {
+                comp = mount(<TestComponent />, { attachTo });
+                const node = ReactDOM.findDOMNode(comp.instance());
+
+                dispatchTestKeyboardEvent(node, "keyup", "1");
+                expect(localKeyUpSpy.called).to.be.true;
+
+                dispatchTestKeyboardEvent(node, "keyup", "2");
+                expect(globalKeyUpSpy.called).to.be.true;
+            });
+
+            it("triggers only global hotkey when not focused", () => {
+                comp = mount(<div><TestComponent /><div className="unhotkeyed" tabIndex={2} /></div>, { attachTo });
+                const unhotkeyed = ReactDOM.findDOMNode(comp.instance()).querySelector(".unhotkeyed");
+                (unhotkeyed as HTMLElement).focus();
+
+                dispatchTestKeyboardEvent(unhotkeyed, eventName, "1");
+                expect(getLocalSpy(eventName).called).to.be.false;
+
+                dispatchTestKeyboardEvent(unhotkeyed, eventName, "2");
+                expect(getGlobalSpy(eventName).called).to.be.true;
+            });
+
+            it("ignores hotkeys when disabled={true}", () => {
+                comp = mount(<TestComponent disabled={true} />, { attachTo });
+                const node = ReactDOM.findDOMNode(comp.instance());
+
+                dispatchTestKeyboardEvent(node, eventName, "1");
+                expect(getLocalSpy(eventName).called).to.be.false;
+
+                dispatchTestKeyboardEvent(node, eventName, "2");
+                expect(getGlobalSpy(eventName).called).to.be.false;
+            });
+
+            it("prevents default if preventDefault={true}", () => {
+                comp = mount(<TestComponent preventDefault={true} />, { attachTo });
+                const node = ReactDOM.findDOMNode(comp.instance());
+
+                dispatchTestKeyboardEvent(node, eventName, "1");
+                const localEvent = getLocalSpy(eventName).lastCall.args[0] as KeyboardEvent;
+                expect(localEvent.defaultPrevented).to.be.true;
+
+                dispatchTestKeyboardEvent(node, eventName, "2");
+                const globalEvent = getGlobalSpy(eventName).lastCall.args[0] as KeyboardEvent;
+                expect(globalEvent.defaultPrevented).to.be.true;
+            });
+
+            // TODO (clewis): we need to get the following test working to actually verify the
+            // stopPropagation behavior, but it's proving difficult.
+            it.skip("stops propagation if stopPropagation={true}", () => {
```

Comment:
@giladgray `isPropagationStopped` is only a jQuery thing. https://stackoverflow.com/questions/7259753/in-a-javascript-event-how-to-determine-stoppropagation-has-been-called